website_multi_theme/models/website.py

```diff
@@ -202,9 +202,13 @@ def _multi_theme_activate(self):
             views_to_remove = website.multi_theme_view_ids - custom_views
             # Removed views could be a copied parent for others
             # So, replace to original parent first
-            for view in self.env['ir.ui.view'].search([
-                    ('inherit_id', 'in', views_to_remove.ids)
-            ]):
+            views_to_replace_parent = \
+                self.env['ir.ui.view']\
+                    .with_context(active_test=False)\
```

Comment:
While it may confuse when to use and when don't use ``active_test`` in this module. But in this case is easier: we must replace parent for all views because otherwise, we get constraint error (deleted record is still referenced), Postgres constraint doesn't care about ``active`` field value